web/modules/custom/hoeringsportal_hearing/hoeringsportal_hearing.module

```diff
@@ -44,3 +46,37 @@ function hoeringsportal_hearing_form_alter(&$form, FormStateInterface $form_stat
     }
   }
 }
+
+/**
+ * Implements hook_ENTITY_TYPE_view() for node entities.
+ */
+function hoeringsportal_hearing_node_view(array &$build, EntityInterface $entity, EntityViewDisplayInterface $display, $view_mode) {
+  if ($entity->getType() == 'hearing') {
+    $view_mode_change_title = [
+      'full',
+      'hearing_ticket_add',
+      'hearing_ticket_view',
+
+    ];
+    $content_state = $entity->field_content_state->value;
+    if (in_array($view_mode, $view_mode_change_title)) {
+      switch ($content_state) {
+        case 'upcoming':
+          $start_date = $entity->field_start_date->date->format('d/m/Y');
```

Comment:
Why are we using `field_start_date` here?

Reply:
That was a feature request from designer